File: CiscoPCCE.Toolkit/Bean/AtomLink.cs
using System.Xml.Serialization;

using CiscoPCCE.Toolkit.Sdk;

namespace CiscoPCCE.Toolkit.Bean
{
    public class AtomLink : BaseApiBean
    {
        [XmlElement("href")]
        public string? Href { get; set; }

        [XmlElement("rel")]
        public string? Rel { get; set; }

        [XmlElement("type")]
        public string? Type { get; set; }
    }
}

File: JavaSrc/toolkit/bean/AtomLink.java
// ----------------------------------------------
package com.cisco.ccbu.cce.unifiedconfig.toolkit.bean;

import com.cisco.ccbu.cce.unifiedconfig.toolkit.BaseApiBean;
import javax.xml.bind.annotation.XmlSeeAlso;
import javax.ws.rs.Path;
import javax.xml.bind.annotation.XmlRootElement;
import java.util.List;
import com.cisco.ccbu.cce.unifiedconfig.toolkit.ReferenceBean;
import java.util.Date;
import com.cisco.ccbu.cce.unifiedconfig.toolkit.BaseApiListBean;

import javax.xml.bind.annotation.XmlElement;

import javax.xml.bind.annotation.XmlElementWrapper;


public class AtomLink extends BaseApiBean {
  private String href;
  private String rel;
  private String type;

  public String getHref() {
     return this.href;
  }

  public void setHref(String href) {
     this.href = href;
  }

  public String getRel() {
     return this.rel;
  }

  public void setRel(String rel) {
     this.rel = rel;
  }

  public String getType() {
     return this.type;
  }

  public void setType(String type) {
     this.type = type;
  }


}
